pyproject.toml

```diff
@@ -39,15 +53,27 @@ pytest-asyncio = "^0.18.3"
 pytest-cov = "^3.0.0"
 liccheck = "^0.7.2"
 pytest-mock = "^3.10.0"
+isort = "^5.11.4"
 
 [tool.poetry.scripts]
 dapp-runner = "dapp_runner.__main__:_cli"
 
-[[tool.poetry.source]]
-name = "testpypi"
-url = "https://test.pypi.org/simple/"
-default = false
-secondary = true
+[tool.poe.tasks]
+checks = {sequence = ["checks_codestyle", "checks_typing", "checks_license"], help = "Runs all available code checks"}
```

Comment:
It's usual for the help strings (and generally for any documentation strings) to use the imperative mood  (`Run all ... `) instead of indicative (`Runs all...`)